controller/src/main/java/org/jboss/as/controller/operations/global/GlobalOperationHandlers.java

```diff
@@ -255,33 +258,40 @@ public void handleResult(OperationContext.ResultAction resultAction, OperationCo
         }
 
         /**
-         * Wraps a call to {@link #execute(org.jboss.as.controller.PathAddress, org.jboss.as.controller.PathAddress, org.jboss.as.controller.OperationContext, boolean)}
+         * Wraps a call to
+         * {@link #execute(org.jboss.as.controller.PathAddress, org.jboss.as.controller.PathAddress, org.jboss.as.controller.OperationContext, org.jboss.as.controller.registry.ImmutableManagementResourceRegistration, boolean)}
          * with logic to handle authorization and missing resource exceptions.
          *
-         * @param address the full address of the original request
          * @param base the portion of the address that has already been processed
+         * @param remaining the unprocessed portion of the original address
          * @param context  context of the request
+         * @param registration the resource registration for the {@code base} address
          * @param ignoreMissing {@code true} if this is being called for an "optional" portion of an address tree
          *                                  and NoSuchResourceException should be ignored. A portion of an address
          *                                  tree is optional if it is associated with a wildcard or multi-segment
          *                                  element in {@code address}
          */
-        private void safeExecute(final PathAddress address, final PathAddress base, final OperationContext context, boolean ignoreMissing) {
+        private void safeExecute(final PathAddress base, final PathAddress remaining, final OperationContext context,
+                final ImmutableManagementResourceRegistration registration, boolean ignoreMissing) {
             try {
-                execute(address, base, context, ignoreMissing);
+                ControllerLogger.MGMT_OP_LOGGER.tracef("safeExecute for %s, remaining is %s", base, remaining);
+                execute(base, remaining, context, registration, ignoreMissing);
             } catch (UnauthorizedException e) {
                 // equivalent to the resource not existing
                 // Just report the failure to the filter and complete normally
                 filteredData.addReadRestrictedResource(base);
+                ControllerLogger.MGMT_OP_LOGGER.tracef("Caught UnauthorizedException in %s", this);
```

Comment:
Should `toString()` be overridden so when `this` is logged it's something more meaningful than the default `toString()`?